cowboy_erlydtl.template

```diff
@@ -8,8 +8,9 @@
 %% base
 
 {template, "template/dot.gitignore", "{{name}}/.gitignore"}.
-{template, "template/dtl.gitignore", "{{name}}/priv/templates-compiled/.gitignore"}.
+{template, "template/dot.gitkeep", "{{name}}/priv/templates-compiled/.gitkeep"}.
```

Comment:
The top-level `.gitignore` takes care of the compiled template files, however, the directory must be present so the zero-byte `.gitkeep` file is created. There's nothing special about the name, it's just a widely-used convention to preserve an empty directory in a git repo.